cmd/ipfs-cluster-follow/commands.go

```diff
@@ -486,6 +486,57 @@ func listCmd(c *cli.Context) error {
 	return nil
 }
 
+// Stop
+func stopCmd(c *cli.Context) error {
+	clusterName := c.String(clusterNameFlag)
+
+	absPath, configPath, identityPath := buildPaths(c, clusterName)
+	if !isInitialized(absPath) {
+		printNotInitialized(clusterName)
+		return cli.Exit("", 1)
+	}
+
+	// unpin cluster files and kill ipfs-cluster-follow process if cluster and ipfs daemon exist
+	_, err := os.Stat(filepath.Join(absPath, "api-socket"))
+	if err == nil {
+		err = unpinEverything(absPath, clusterName, identityPath)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "unpin error"), 1)
+		}
```

Comment:
I think unpin should be behind a flag.

- `ipfs-cluster-follow myCluster stop`   should just kill the cluster peer
- `ipfs-cluster-follow myCluster stop --unpin --cleanup` should perform the unpin and cleanup actions.

The reason is that I wouldn't expect a stop command to be destructive by default (deleting pins).

Reply:
fix, but maybe `ipfs-cluster-follow myCluster stop --cleanup` will not unpin everything but only delete myCluster folder?